installer/rpm/mongodb-database-tools.spec

```diff
@@ -8,7 +8,7 @@ Group: Applications/Databases
 License: Apache
 URL:        http://www.mongodb.com
 Vendor:     MongoDB
-Requires: openssl, cyrus-sasl, cyrus-sasl-plain, cyrus-sasl-gssapi
+Requires: cyrus-sasl, cyrus-sasl-plain, cyrus-sasl-gssapi
```

Comment:
So we're removing the sasl tag which would link to libsasl, but we still require these sasl libs, is that right?

Reply:
I was trying to be conservative here because I'm  not sure if we really need them. But looking at this more closely I suspect we can remove them.

Reply:
I realized that I could test this. I ran a RHEL 8 Docker image locally and installed the tools, along with all of the above packages (including `openssl`). Then I forcibly deleted all the files for these libraries. The installed tools binaries continued to run without any linker issues. So I will remove all of these dependencies.